### ISSUE TITLE
Tracking page view not complete since 5.0.1

### ISSUE THREAD

**Issue by Substitut** [EmmanuelRoux/ngx-matomo#67]
Hello,

I'm having strange issue the 5.0.1 migration, I'm only having the first page tracked into my matomo dashboard.

Below it's my initialization code :

```ts
NgxMatomoModule.forRoot({
      mode: MatomoInitializationMode.AUTO_DEFERRED,
      requireConsent: MatomoConsentMode.COOKIE,
      disabled: isDevelopment(),
    }),
    NgxMatomoRouterModule,
```

```ts
     if (response.body) {
        matomoInitializer.initializeTracker({
          siteId: response.body.matomoId,
          trackerUrl: getMatomoTrackerUrl()
        })
      }
```

As I said I still see my first event but not more. I'm using tracker for event tracking in some component also.
Am I the only one facing this issue ?

**Comment by EmmanuelRoux** [comment]
Hi @Substitut 

Can you please share a repro of your code? (Example: https://stackblitz.com/edit/stackblitz-starters-pagumy )

It may be a bug, but also some problems with your environment/config. Do you call `setCookieConsentGiven` at some point?

**Comment by Substitut** [comment]
Not exactly as I'm using `this.matomo.rememberCookieConsentGiven();` and I can see in my cookies MATOMO_SESSID, _pk_id.6.5ba2, mtm_cookie_consent, .. So it seems cookies are properly applied. I looking to provide you a sample that reproduces the issue.

I tried to disable requireConsent `MatomoConsentMode.NONE` and still the same, only the first page loaded is sent.
![Capture d’écran 2023-07-25 à 11 57 11](https://github.com/EmmanuelRoux/ngx-matomo/assets/1788073/fe4975c9-0b75-44be-86a5-97a6c32ceab8)

I did some navigation, I only see the new page when I reload the application.  And others navigation are lost.

**Comment by EmmanuelRoux** [comment]
Hi @Substitut 

Here are working basic examples with `ngx-matomo-client@5.0.1`:
https://stackblitz.com/edit/stackblitz-starters-6dhvtn
(or here for standalone components: https://stackblitz.com/edit/stackblitz-starters-mgqc8e)

**Comment by EmmanuelRoux** [comment]
Oh you're right! You must call `.forRoot()` on `NgxMatomoRouterModule`!
So your configuration should look like this:
```ts
NgxMatomoModule.forRoot({
      mode: MatomoInitializationMode.AUTO_DEFERRED,
      requireConsent: MatomoConsentMode.COOKIE,
      disabled: isDevelopment(),
    }),
    NgxMatomoRouterModule.forRoot(),
```

Since `ngx-matomo-client@5.0.0` it is indeed required to call `.forRoot()`. But this is not mentioned in documentation and not handled in `ng add` schematics. I will fix it soon.

In the meantime calling `.forRoot()` should fix your issue, please let me know!

**Comment by Substitut** [comment]
It's working like a charm now ! It was driving me crazy .

**Comment by EmmanuelRoux** [comment]
Fixed in `v5.0.2`, see #68
